### PR TITLE
Unit test for NLU model load and parse

### PR DESCRIPTION
**Pull request by kavyasrinet**
# Description

This PR does two things:
1. Adds in a unit test to make sure NLU model loads and performs inference in the current environment. This has caused some regressions lately - adding in a test.
2. Variable renaming in some files - TTAD was a very old convention.

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [x] This is a unit test
- [x] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [x] I want a high level review. 
- [ ] I want a deep design review.

# Testing

Tested locally, on devfair and CI

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I have verified that all scripts in `tests/scripts` runs on hardware.

### CODE REVIEW

**Comment by kavyasrinet** [comment]
Oh no, just realized that I forgot to add reviewers to this :(

File: droidlet/perception/semantic_parsing/tests/test_nsp_loading.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class TestNSPModel(unittest.TestCase):
+    def setUp(self):
+        nsp_data_dir = NLU_DATA_DIR
+        nsp_model_dir = os.path.join(NLU_MODEL_DIR, "ttad_bert_updated")
+        self.model = Model(model_dir=nsp_model_dir, data_dir=nsp_data_dir)
+
+    def test_model_parse(self):
+        chat = 'come here'
+        logical_form = self.model.parse(chat=chat)
+        self.assertEqual(type(logical_form), dict)
```
Comment (anuragprat1k) [inline droidlet/perception/semantic_parsing/tests/test_nsp_loading.py:22]: Couple of thoughts here:
1. Is there something more than a type check we can do here? 
2. Can you also add a test for a case when model loading throws an exception - for eg when model_dir is wrong / empty etc.

**Review by anuragprat1k — CHANGES_REQUESTED** [review]
Good idea! Requesting a tiny change.

**Comment by kavyasrinet** [comment]
@anuragprat1k can you re-review please ?

File: droidlet/perception/semantic_parsing/tests/test_nsp_loading.py (hunk 1 above)
Comment (anuragprat1k) [reply droidlet/perception/semantic_parsing/tests/test_nsp_loading.py:21]: Yea, model testing is tricky. I thought just type check might not be sufficient because it would pass even when an empty dict is returned. If an empty dict means that the NSP is broken in some way, then we should add a stronger check (since we wouldn't want this test to pass then). If not, then this is fine.

**Review by anuragprat1k — APPROVED** [review]

File: droidlet/perception/semantic_parsing/tests/test_nsp_loading.py (hunk 1 above)
Comment (kavyasrinet) [reply droidlet/perception/semantic_parsing/tests/test_nsp_loading.py:21]: Yeah I added a stricter check to all three : `self.assertTrue("dialogue_type" in logical_form)` since we expect "dialogue_type" key in the dict at the minimum
